### ISSUE TITLE
R summary estimate using bpmodels

### ISSUE THREAD

**Issue by adamkucharski** [epiverse-trace/superspreading#6]
Given data on early transmission clusters, would be useful to have an initial estimate of R with uncertainty, as well as an estimate of P(R>1). Below gives a simple implementation using a profile likelihood with `quickfit` (there may be a more stable efficient package for this, however).

Calculating P(R>1) would require doing the inverse inference, i.e. updating `quickfit` (or another package) to find the level of confidence that gives an upper bound corresponding to R=1.

```
#devtools::install_github("adamkucharski/quickfit")
#devtools::install_github('sbfnk/bpmodels')
library(quickfit)
library(stats4)
library(bpmodels)

# Early clusters
cluster_sizes <- c(rep(1,10),5,10)

# Estimate R and k with quickfit
log_l <- function(x,a,b){
  bpmodels::chain_ll(x,"nbinom",stat="size",
                     mu=a,size=b) 
}

estim <- quickfit::calculate_profile(log_l,
                            data_in=cluster_sizes,
                            n_param=2,
                            a_initial = 0.5,
                            b_initial = 1,
                            precision=0.01)

# Reformat and output CI for R:
estim <- data.frame(t(unlist(estim)))

c(estim$estimate.a,estim$profile_out.a1,estim$profile_out.a2)
```

**Comment by adamkucharski** [comment]
This is perhaps better placed as a vignette with {bpmodels}? Or an extension example linked to issue #16 ?

**Comment by jamesmbaazam** [comment]
It does seem like the core engine here is from {bpmodels}, so better linked to the original package. #16 is a full pipeline and that's where I get confused about whether it belongs in {episoap}?

**Comment by joshwlambert** [comment]
Yes, this can now be closed. I don't think this fits in with {episoap} or a vignette on {epichains}, and like you say @chartgerink {bpmodels} is now retired. It could potentially be added as a [howto](https://github.com/epiverse-trace/howto). 

Tagging @adamkucharski and @avallecam to see if it would be useful to open an issue in the howto repo.
